packages/collector/test/tracing/opentelemetry/tedious-app.js

```diff
@@ -52,14 +52,14 @@ const config = {
   },
   options: {
     database: azureConfig?.AZURE_SQL_DATABASE || process.env.AZURE_SQL_DATABASE,
-    connectTimeout: 20000
+    connectTimeout: 30000
   }
 };
 
 let connected = false;
 let connection;
 
-const retryDelay = 1000;
+const retryDelay = 30000;
```

Comment:
Q: I don't understand this change 🤔 

The retry mechanism helps to retry connecting to Azure. Why would we wait 30s on each retry?
From my understanding the initial run does not even trigger the retry logic.

Does this change help to fix the problem? I thought the `connectionTimeout` and modifying `await controls.startAndWaitForAgentConnection` solved the problem.

Reply:
This has to be updated because we have to give some time for the DB to get online. Since we are retrying only 2 times, this 30-second delay won't be an issue. 
This might work with a 20 or 25 decond delay. but as a safer I put it for 30 seconds

Reply:
The initial connection request triggers the retry after the connectionTimeout mentioned. 
Even the connection timeout can also be kept as 20 sec, but increased for safety.

Reply:
But you told me that the retry mechanmisn is not even triggered? 🤔 
And I haven't seen the logs as well:

```sh
Connected to the database
[ProcessControls] server is up.
[ProcessControls] started
```

> setTimeout(connectWithRetry, retryDelay);

Is called after

> connection.on('connect', err => {

Thats why I am confused by this change.

Reply:
Ah okay 👍 Thanks for explaining.